src/main/charts/bamboo/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+name: bamboo
+description: A chart for installing Bamboo Data Center on Kubernetes
+type: application
+version: 0.0.1 # TODO: THIS NEEDS TO BE CONFIRMED 
+appVersion: 8.0.1-jdk11  # TODO: THIS NEEDS TO BE CONFIRMED 
+kubeVersion: ">=1.19.x-0"
+keywords:
+  - Bamboo
+  - Bamboo Server
+  - Bamboo Data Center
+  - Bamboo DC
+  - Atlassian
+home: https://www.atlassian.com/software/bamboo
+icon: https://atlassian-labs.github.io/data-center-helm-charts/icons/bamboo.svg
+sources:
+  - https://github.com/atlassian-labs/data-center-helm-charts
+  - https://bitbucket.org/atlassian-docker/docker-bamboo-server
+  - https://bitbucket.org/atlassian-docker/docker-bamboo-agent-base
```

Comment:
These will likely be updated once @tarka has merged his work

Reply:
These will remain correct, I'm porting to work into those repos.